File: codigos algoritmicos python/ejercicio20.py
print("validar usuatrio y contraseña")
usuario= input("ingrese usuario:  ")
contraseña= input("ingrese contraseña:  ")
if(usuario=="admin" and contraseña=="1234"):
    print("acceso concedido")
elif(usuario!="admin" and contraseña=="1234"):
        print("usuario incorrecto")
elif(usuario=="admin" and contraseña!="1234"):
        print("contraseña incorrecta")
else:
        print("usuario y contraseña incorrectos")
        

File: codigos algoritmicos javascript/ejercicio20.js
console.log("validar usuario y contraseña");
usuario="admin";
contraseña="123456";
if (usuario=="admin" && contraseña=="123456"){
    console.log("Acceso concedido");
}
else if (usuario!="admin"){
    console.log("Usuario incorrecto");
}
else{
    console.log("Contraseña incorrecta");
}
